packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -341,7 +341,8 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
         isEventTargetOutsideCallout &&
         ((this._target as MouseEvent).stopPropagation ||
           !this._target ||
-          (target !== this._target && !elementContains(this._target as HTMLElement, target))))
+          (target !== this._target &&
```

Comment:
Does this condition need to be:
`(this.props.allowDismissOnTargetClick || target !== this._target && !elementContains(this._target as HTMLElement, target))` so that it also dismisses when `target === this._target`?

Reply:
Ah, that makes sense. Thanks for the catch!